src.ts/account.ts

```diff
@@ -10,10 +10,10 @@ import { PublicKey } from './utils/key_pair';
 import { PositionalArgsError } from './utils/errors';
 import { parseRpcError } from './utils/rpc_errors';
 
-// Default amount of tokens to be send with the function calls. Used to pay for the fees
+// Default amount of gas to be sent with the function calls. Used to pay for the fees
 // incurred while running the contract execution. The unused amount will be refunded back to
 // the originator.
-const DEFAULT_FUNC_CALL_AMOUNT = 2000000 * 1000000;
+const DEFAULT_FUNC_CALL_GAS = "100000000000000000";
```

Comment:
This is more than `max_gas_burnt` (2 * 10^14) so probably not a good idea.

Reply:
So what is gas measured in? How much NEAR is max-gas-burnt?

Reply:
So what is gas measured in? How much NEAR is max-gas-burnt?

Reply:
Actually I `max_gas_burnt` is not the most relevant thing here since a call can generate receipts that span multiple blocks. So I think we should use `max_prepaid_gas` which is 10^16

Reply:
@nearmax any concerns re setting this to `max_prepaid_gas`? Seems like with current gas price it's around 0.00001 NEAR.